CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```

Comment:
Optional: In my opinion, we should include a link to our Code of Conduct file here